tests/flow/test_index_create.py

```diff
@@ -185,3 +185,60 @@ def create_drop_index(graph_id):
                 redis_con.execute_command("GRAPH.DELETE", f"x{graph_id}")
         pool = Pool(nodes=10)
         pool.map(create_drop_index, range(1, 100))
+
+    def test06_syntax_error_index_creation(self):
+        # create index on invalid property name
```

Comment:
do we handle now?
```
CREATE INDEX FOR (p:Person) ON (z.q)
```

Reply:
Hi,
No. The validation of that case has not been implemented.
The PR only includes validations in the parser, and for that case the validation should be done in Redisgraph.
I think that validation of undefined variables should be similar to the method `static AST_Validation _Validate_Aliases_DefinedInClause(const cypher_astnode_t *clause,
rax *defined_aliases)`, Is that the correct approach?
Thanks.

Reply:
no we don't get to this part of the code I think better to validate it in https://github.com/RedisGraph/RedisGraph/blob/51df7d0310c9ef5a76496b158c90be067a6ed080/src/ast/ast_validations.c#L1844

Reply:
Thanks.  The validation of undefined identifiers was included in [246ffb1](https://github.com/RedisGraph/RedisGraph/pull/2597/commits/246ffb106949cb00d075c87e1b7e86ea11c34026)